.github/actions/docs/action.yaml

```diff
@@ -43,18 +43,18 @@ runs:
       shell: bash -l {0}
       run: |
         sdk install java 8.312.07.1-amzn
-        sdk install scala 2.13.0
+        sdk install scala 2.13.14
         sdk install sbt 1.10.4
-    - name: Cache .ivy2
-      uses: actions/cache@v4
-      with:
-        path: ~/.ivy2
-        key: ${{ runner.os }}-ivy2-${{ hashFiles('**/build.sbt') }}
-    - name: Cache .sbt
-      uses: actions/cache@v4
-      with:
-        path: ~/.sbt
-        key: ${{ runner.os }}-sbt-${{ hashFiles('**/build.properties') }}
+#    - name: Cache .ivy2
+#      uses: actions/cache@v4
+#      with:
+#        path: ~/.ivy2
+#        key: ${{ runner.os }}-ivy2-${{ hashFiles('**/build.sbt') }}
+#    - name: Cache .sbt
+#      uses: actions/cache@v4
+#      with:
+#        path: ~/.sbt
+#        key: ${{ runner.os }}-sbt-${{ hashFiles('**/build.properties') }}
```

Comment:
Can we turn these back on? If you'd like to try something simpler, this PR has a simpler Java/Scala setup:

https://github.com/fulcrumgenomics/fgbio/pull/1057/files

Reply:
that's emberassing